manage.py

```diff
@@ -1,13 +1,5 @@
 #!/usr/bin/env python3
 
-import os
-import sys
+from evap.__main__ import main
 
-if __name__ == "__main__":
-    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "evap.settings")
-
-    from django.conf import settings
-    from django.core.management import execute_from_command_line
-
-    settings.DATADIR.mkdir(exist_ok=True)
-    execute_from_command_line(sys.argv)
+main()
```

Comment:
Not that anyone would ever import this file, but is there a specific reason to remove the `if __name__ == "__main__"` guard?

Reply:
No reason except brevity

Reply:
I can add it back if you want

Reply:
`runpy` sure is ... something :D It fixes the missing coverage, but it adds some complexity that we might not want. @richardebeling ?

Reply:
Looks like the right thing to do to me. I'm not sure I get your points about missing coverage (you mean test coverage? Would that have been a problem?) and complexity (import runpy and runpy.run_module look canonical to me, I wouldn't care what they do under the hood -- do I need to care?), unless I'm missing something here I'm very fine with the current code

Reply:
I was wondering whether `runpy` magic could break something like `./manage.py test --pdb`, but it seems to work just fine. My points are
- without runpy, the line in `__main__.py` that calls `main` is not covered, but with runpy, it is (although the branch where `__name__` is _not_ `__main__` is uncovered now :D)
- I also don't care what runpy does and don't want to care, I was just afraid that we would have to care some day. It also makes the `manage.py` file less approachable to first semester students, but I guess they have to accept our arcane magic anyways once they hit `execute_from_command_line`

I also think runpy is fun, so we can keep it if the above don't concern you :)

Reply:
Alright, makes sense. I'm fine with both approaches.